java/sage/MediaServer.java

```diff
@@ -1157,6 +1213,147 @@ else if (tempString.indexOf("XCODE_SETUP ") == 0 && !Sage.EMBEDDED)
             int numWritten = s.write(commBufWrite);
             if (MEDIA_SERVER_DEBUG) System.out.println("MediaServer wrote out " + numWritten + " bytes");
           }
+          else if (tempString.indexOf("REMUX_SETUP ") == 0)
+          {
+            // REMUX_SETUP Mode OutputFormat Parameters...
+            // Auto:
+            // REMUX_SETUP AUTO OutputFormat IsTV
+            // Ex. REMUX_SETUP AUTO PS TRUE
+
+            // The file must be closed to turn this mode off.
+            if (remuxer != null)
+            {
+              commBufWrite.clear();
+              commBufWrite.put("INIT_ERROR\r\n".getBytes()).flip();
+
+              if (Sage.DBG) System.out.println("MediaServer is already in remux mode ignoring: " +
+                  tempString.substring(tempString.indexOf(" ") + 1));
+            }
+            else
+            {
+              if (Sage.DBG) System.out.println("MediaServer is writing in remux mode: " +
+                  tempString.substring(tempString.indexOf(" ") + 1));
+
+              StringTokenizer toker = new StringTokenizer(tempString.substring(12), " ");
+
+              String mode = toker.nextToken();
+              int outputFormat = toker.nextToken().equals("TS") ? MPEGParser2.REMUX_TS : MPEGParser2.REMUX_PS;
```

Comment:
You may want to use equalsIgnoreCase for these parameter tests...especially the one for TRUE since somebody may just end up printing out a boolean value into the text at some point.